refinery/file_store/models.py

```diff
@@ -240,11 +239,9 @@ class FileStoreItem(models.Model):
     import_task_id = UUIDField(auto=False, blank=True)
     # Date created
     created = models.DateTimeField(auto_now_add=True,
-                                   default=timezone.now,
                                    blank=True)
```

Comment:
`blank=True` is not needed either:
"As currently implemented, setting auto_now or auto_now_add to True will cause the field to have editable=False and blank=True set"
https://docs.djangoproject.com/en/1.8/ref/models/fields/#datefield